src/stores/SpaceStore.tsx

```diff
@@ -41,6 +41,9 @@ type SpaceKey = string | symbol;
 interface IState {}
 
 const ACTIVE_SPACE_LS_KEY = "mx_active_space";
+const LAST_VIEWED_ROOMS = "mx_last_viewed_rooms";
+
+const LAST_VIEWED_ROOMS_HOME = "home_space";
 
 export const HOME_SPACE = Symbol("home-space");
```

Comment:
Should I try to use the `Symbol`?

Reply:
if you're putting the room id in the localStorage keys you could do 

`mx_......_${s === HOME_SPACE ? "home" : s.roomId)` or similar

Reply:
> `mx_......_${s === HOME_SPACE ? "home" : s.roomId} or similar`

I am already doing something like this:

```ts
const activeSpaceId = this.activeSpace?.roomId || LAST_VIEWED_ROOMS_HOME;                
window.localStorage.setItem(`${LAST_VIEWED_ROOMS}_${activeSpaceId}`, payload.room_id);
```

Am I missing something?